images/lisk/files/home/lisk/wait-for-it.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
Please, may you explain the main reason of this script?

Reply:
This script is used to wait for the database to be reachable; see also https://docs.docker.com/compose/startup-order/